themes-default/slim/static/js/store.js

```diff
@@ -270,7 +306,7 @@ const store = new Puex({
             if (section === 'main') {
                 state.config = config;
             }
-            if (['qualities', 'statuses'].includes(section)) {
+            if (['qualities', 'statuses', 'metadata'].includes(section)) {
```

Comment:
mehh, this is not so nice.
@OmgImAlexis 
Took me a while to notice it. Any chance this can be improved?

Reply:
@sharkykh any ideas here?

Reply:
I feel like `includes` makes more sense?

Reply:
I was thinking more in separating the list of allowed section from the code. Or make it more obvious. Maybe use a constant at top of the file?

Reply:
```js
const includesSection = () => ['qualities', 'statuses', 'metadata'].includes(section);
if (includesSection) {}
```

Reply:
Like that?